perfkitbenchmarker/publisher.py

```diff
@@ -640,6 +650,81 @@ def _deDotKeys(self, res):
     return res
 
 
+class InfluxDBPublisher(SamplePublisher):
+
+  def __init__(self, influx_uri=None, influx_db_name=None):
+    # set to default above in flags unless changed
+    self.influx_uri = influx_uri
+    self.influx_db_name = influx_db_name
+
+  def PublishSamples(self, samples):
+    successful_http_request_codes = [200, 202, 204]
+    sample = list(samples)
+    number_of_samples = 0
+    for sample in samples:
+      number_of_samples += 1
+      timestamp = int((10 ** 9) * sample['timestamp'])
+      measurement = 'perfkitbenchmarker'
+      tag_set_metadata = \
```

Comment:
nitpick: Prefer hanging indentation for style consistency of the codebase. See: https://www.python.org/dev/peps/pep-0008/#indentation